Code/Tools/ProjectManager/Source/ProjectSettingsScreen.cpp

```diff
@@ -60,6 +60,10 @@ namespace O3DE::ProjectManager
         connect(m_projectPath->lineEdit(), &QLineEdit::textChanged, this, &ProjectSettingsScreen::OnProjectPathUpdated);
         m_verticalLayout->addWidget(m_projectPath);
 
+        m_projectId = new FormLineEditWidget(tr("Project ID"), "", this);
+        connect(m_projectId->lineEdit(), &QLineEdit::textChanged, this, &ProjectSettingsScreen::OnProjectIdUpdated);
+        m_verticalLayout->addWidget(m_projectId);
```

Comment:
Looking at the UX, I think it might be better not to show this to users initially because it raises questions that may confuse most of our users - alternately, perhaps we create a collapsible "Advanced" section and put it in there.  Ideally, 99% of users won't need or want to change this value.

Reply:
I will chat with UX over this. I would say I agree we probably don't need the UX at all on the Create Project screen after thinking about it. I still think we should allow the user to change it somehow through the Edit Project Screen though.